### ISSUE TITLE
How's your job going on?

### ISSUE THREAD

**Issue by chaseliutb** [HtutLynn/Pytorch_EfficientDet_to_TensorRT#2]
I'm also wondering how to convert pretrained efficientdet to tensorRT to accelerate the inference work..

**Comment by HtutLynn** [comment]
It seems to be that padding operation used in efficientdet implementation is causing the problem in TensorRT conversion. I opened a [question thread](https://forums.developer.nvidia.com/t/unable-to-convert-onnx-model-to-tensorrt/142431) on Nvidia TensorRT forum so it's better to go check there for further info. I created this repo to test if I can convert the model to TensorRT by making everything static. For example, creating separate implementations for EfficientDet0-7. But no progress yet.
